### PR TITLE
Collapse code & add text page

### PR DESCRIPTION
**Pull request by t-lji**
1. add a feature to collapse code block
2. change the icon instead of popping up a alert after copied code successfully
3. add a new page to introduce @fluentui/text

### CODE REVIEW

**Comment by t-lji** [comment]
collapse copy

This gif displaying:
1. feat: collapse code block. You can collapse or expand code block by clicking the Button
2. feat: change copy code icon. After clicking Button, you can copy the code to your clipboard and use it in anywhere and the Button will recover to default after 1 second.

**Comment by t-lji** [comment]
component_text

This gif showing ui/component/text page, I add some @fluentui/text example to help developer add text component easily.

File: tabs/src/components/code/component/text/Toolbar.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { Toolbar } from '@fluentui/react-northstar'
+import { BoldIcon, ItalicIcon, UnderlineIcon, FontSizeIcon, RemoveFormatIcon } from '@fluentui/react-northstar'
+import { OutdentIcon, IndentIcon, MoreIcon, LinkIcon, CodeSnippetIcon, QuoteIcon } from '@fluentui/react-northstar'
```
Comment (SLdragon) [inline tabs/src/components/code/component/text/Toolbar.jsx:4]: They are all from @fluentui/react-northstar, so you can make them in one import sentence

File: tabs/src/components/ui/Component.jsx [hunk 2]
```diff
@@ -1,11 +1,26 @@
 import React from "react";
+import { Header } from '@fluentui/react-northstar'
+import { Link } from 'react-router-dom'
+import "./Overall.css";
```
Comment (SLdragon) [inline tabs/src/components/ui/Component.jsx:4]: for best practice, css file name should use lowercase string

File: tabs/src/components/code/component/text/Toolbar.jsx (hunk 1 above)
Comment (t-lji) [reply tabs/src/components/code/component/text/Toolbar.jsx:4]: ok, I will add them in one import sentence and split them to lines

File: tabs/src/components/ui/component/Text.jsx [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'remove-format',
+            title: 'Remove formatting',
+          },
+          {
+            key: 'divider-2',
```
Comment (SLdragon) [inline tabs/src/components/ui/component/Text.jsx:243]: Did this key required?

File: tabs/src/components/ui/Component.jsx (hunk 2 above)
Comment (t-lji) [reply tabs/src/components/ui/Component.jsx:4]: thanks, I will rename the css file.

File: tabs/src/components/ui/component/Text.jsx (hunk 3 above)
Comment (t-lji) [reply tabs/src/components/ui/component/Text.jsx:243]: Thanks, this key is not required, I will remove it.

**Review by SLdragon — APPROVED** [review]

**Pull request merged**